src/pybind/mgr/dashboard/frontend/src/app/core/create-cluster-wizard/create-cluster-wizard.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="container h-75">
+  <div class="row h-100 justify-content-center align-items-center">
+    <div class="blank-page">
+      <h2 class="blank-page-title"
+          i18n>
+          Please create your first cluster
```

Comment:
Can you adapt this to conform to [i18n suggestions](https://docs.ceph.com/en/latest/dev/developer_guide/dash-devel/#suggestions) ?